tests/test_facade.py

```
@@ -636,9 +698,7 @@ def test_windows_with_internet_settings_rejected(self, tmp_path: Path) -> None:
         """Egress allowlist on Windows guests is Phase 2 scope."""
         disk = tmp_path / "win11.qcow2"
         disk.touch()
-        with pytest.raises(
-            ValueError, match="internet_settings.* not yet supported for Windows"
-        ):
+        with pytest.raises(ValueError, match="internet_settings.* not yet supported for Windows"):
```

Ruff (0.15.14)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)